app/code/Magento/Checkout/Model/DefaultConfigProvider.php

```diff
@@ -397,6 +397,9 @@ private function getQuoteData()
         if ($this->checkoutSession->getQuote()->getId()) {
             $quote = $this->quoteRepository->get($this->checkoutSession->getQuote()->getId());
             $quoteData = $quote->toArray();
+            if (is_object($quote->getExtensionAttributes())) {
```

Comment:
I believe extension attributes can be an object or null, please replace is_object check comparison with null.
Also, please, cover changes with automated tests.

Reply:
Hi @sidolov, I have created additional commit with fixes, that relates to condition "is_object".
As for tests, I didn't find any tests concern operation getting quoteData. 
Thank you.